package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "us-bank-account-validator",
-      "version": "0.2.0",
+      "version": "1.0.0",
```

Comment:
Do we want to change the version number to 1.0.0 for a dependency patch?

Reply:
Wait this is in the `package-lock.json`? We shouldn't be manually editing that anywhere

Reply:
@GoogilyBoogily This wasn't a manual change; it was a done auto-magically by dependabot.